lib/gollum/app.rb

```diff
@@ -355,6 +355,9 @@ class App < Sinatra::Base
         if settings.wiki_options[:template_page] then
           temppage = wiki_page('/_Template')
           @template_page = (temppage.page != nil) ? temppage.page.raw_data : 'Template page option is set, but no /_Template page is present or committed.'
+          if defined?(Gollum::TemplateFilter)
+            @template_page = Gollum::TemplateFilter.filter(@template_page)
```

Comment:
How does this relate to normalization?

Reply:
Shoot, remnant of other PR. 😨

Reply:
But perhaps it doesn't matter, depending on whether you prefer this PR or the one in gollum-lib. :)